lms/djangoapps/mobile_api/social_facebook/groups/views.py

```diff
@@ -0,0 +1,123 @@
+"""
+Views for groups info API
+"""
+
+from rest_framework import generics, status, mixins
+from rest_framework.response import Response
+from ...utils import mobile_view
+from lms.djangoapps.mobile_api.social_facebook.groups import serializers
+import facebook
+
+from django.conf import settings
+_FACEBOOK_API_VERSION = settings.FACEBOOK_API_VERSION
+_FACEBOOK_APP_ID = settings.FACEBOOK_APP_ID
+_FACEBOOK_APP_SECRET = settings.FACEBOOK_APP_SECRET
+
+@mobile_view()
+class Groups(generics.CreateAPIView, mixins.DestroyModelMixin):
+    """
+    **Use Case**
+
+        An API to Create or Delete course groups.
+
+    **Creation Example request**:
+
+        POST /api/mobile/v0.5/social/facebook/groups/<group_id>
```

Comment:
How does the app know which `group_id` to send even before the group is created?  Doesn't the Facebook server determine the `group_id`?